definitions/divorce/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -3381,5 +3381,15 @@
     "TabDisplayOrder": 3,
     "CaseFieldID": "RespondentPcqId",
     "TabFieldDisplayOrder": 30
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.